docs/_includes/engage.html

```diff
@@ -1,6 +1,6 @@
 <!-- engage -->
 <div class="engage">
-    <section class="page-section" id="{{ site.data.sitetext.engage.section | default: "engage" }}">
+    <section class="page-section" style="padding-top: 70px; padding-bottom: 55px;" id="{{ site.data.sitetext.engage.section | default: "engage" }}">
```

Comment:
Move to https://github.com/layer5io/service-mesh-performance/blob/master/docs/_sass/layout/_engage.scss?

Reply:
@leecalcote I tried that in my local. But it doesn't show any changes, rather increases the padding. Can you suggest me some alternative?

Reply:
So, here you are trying to reduce padding explicitly for this element only, if I am correct?
Then, what you can do is that add another class and then add the styles to that class, and then move the position of the definition of that class before or after the other class to make it precedence higher.

Reply:
@Nikhil-Ladha Yup! you're right! & also thanks for the suggestion :smile: